includes/User.php

```diff
@@ -4903,19 +4903,15 @@ private function loadAttributes() {
 	}
 
 	private function compareAttributeValueFromService( $attributeName, $attributeValue ) {
-		if ( !array_key_exists( $attributeName, $this->mOptions ) ) {
-			$this->logAttributeMissing( $attributeName, $attributeValue );
-		} elseif ( $this->mOptions[$attributeName] !== $attributeValue  ) {
-			$this->logAttributeMismatch( $attributeName, $attributeValue );
+
+		// bio and coverPhoto are not used in MW
+		if ( $attributeName == "bio" || $attributeName == "coverPhoto" ) {
```

Comment:
If we expect more attributes like that, I would be good it they were placed in one structure in one place.